.storybook/theme.js

```diff
@@ -0,0 +1,40 @@
+import { create } from '@storybook/theming';
+
+export default create({
+  // Is this a 'light' or 'dark' theme?
+  base: 'dark',
+
+  // Color palette
+  colorPrimary: '#0062ff', // primary color
```

Comment:
Any chance you can use theme tokens from `@carbon/themes` package? The main JavaScript entrypoint has a set of exports of theme tokens.

Reply:
Definitely. The tokens often don't match the naming convention given by storybook so you get things like `secondaryColor: ui-selected` (not our actual secondary color). Using our actual secondary color for the storybook secondary color results in a very different UI than our intended design.

Could definitely use the colors rather than hex values though.